.gitignore

```diff
@@ -15,5 +15,3 @@
 /libpeerconnection.log
 npm-debug.log
 testem.log
-
-.DS_Store
```

Comment:
Er, was that a vote for keeping `.DS_Store` or removing it? :smile: Up to you both.